### DISCUSSION TITLE
Please add this

### DISCUSSION THREAD

**Discussion by SailentWasTaken** [PojavLauncherTeam/PojavLauncher_iOS#250] [Ideas]
Please can you add controller support for PojavLauncher with bluetooth because it’s really hard playing with mobile touch controls on Java.

**Comment by crystall1nedev** [comment]
This is not the correct place to ask for new features. If you wish to request, you should see the Issues tab and file a Feature Request using the templates available. Had you used Issues rather than Discussions you would have found that this has already been asked for.
